airbyte-cdk/python/airbyte_cdk/sources/declarative/declarative_component_schema.yaml

```diff
@@ -427,33 +455,79 @@ definitions:
         type: string
         enum: [OAuthAuthenticator]
       client_id:
+        title: Client ID
+        description: The OAuth client ID
         type: string
+        examples:
+          - "{{ config['client_id }}"
+          - "{{ config['credentials']['client_id }}"
       client_secret:
+        title: Client Secret
+        description: The OAuth client secret
         type: string
+        examples:
+          - "{{ config['client_secret }}"
+          - "{{ config['credentials']['client_secret }}"
       refresh_token:
+        title: Refresh Token
+        description: Credential artifact used to get a new access token
         type: string
+        examples:
+          - "{{ config['refresh_token'] }}"
+          - "{{ config['credentials]['refresh_token'] }}"
       token_refresh_endpoint:
+        title: Token Refresh Endpoint
+        description: Endpoint to send a POST request in order to get a new access token
         type: string
+        examples:
+          - https://connect.squareup.com/oauth2/token
       access_token_name:
+        title: Access Token Field Name
+        description: The name of the field to extract the access token from in the token refresh response
         type: string
         default: "access_token"
+        examples:
+          - access_token
       expires_in_name:
+        title: Expires_in Field Name
+        description: The name of the field the extract the number of seconds the access token will be valid from the token refresh response
         type: string
         default: "expires_in"
+        examples:
+          - expires_in
       grant_type:
+        title: Grant Type
+        description: How the access token is granted
         type: string
         default: "refresh_token"
+        examples:
+          - refresh_token
       refresh_request_body:
+        title: Refresh Request Body
+        description: Body of the request sent to get a new access token
         type: object
         additionalProperties: true
+        examples:
+          -
       scopes:
+        title: Scopes
+        description: List of scopes that should be granted to the access token
         type: array
         items:
           type: string
+        examples:
+          - applicationId: "{{ config['application_id'] }}"
```

Comment:
got the example from [linnworks](https://github.com/airbytehq/airbyte/blob/master/airbyte-integrations/connectors/source-linnworks/source_linnworks/source.py#L65-L69)